python/lsst/ts/pressureSensor/Untitled.ipynb

```diff
@@ -0,0 +1,120 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tm = TransducerModel(\"10.0.1.125\",502)"
```

Comment:
See Tiago's comment on ticket on where notebook should be placed if it's an example that is intended for end-users (if it's a personal test then it shouldn't be committed)... examples are great!
Comments need to be added to assist the user in what might need changing (IPs etc) and any other items of interest.